### ISSUE TITLE
Docker instructions ends with "HTTPS required" as the second step

### ISSUE THREAD

**Issue by vincentbernat** [keycloak/keycloak-web#319]
### Describe the bug

Following the [Docker instructions](https://www.keycloak.org/getting-started/getting-started-docker) to test Keycloak, I am unable to connect to the console as it is said to require HTTPS:

![image](https://user-images.githubusercontent.com/631446/170272751-59b36fb1-75b7-4231-916c-d8de05b333d9.png)

### Version

18.0.0

### Expected behavior

Instructions should explain how to disable SSL or should contain a way to enable SSL with self-signed certificates.

### Anything else?

There are various hints around this in StackOverflow, notably:

```console
$ ./kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin
Logging into http://localhost:8080/auth as user admin of realm master
Enter password:
null [RESTEASY003210: Could not find resource for full path: http://localhost:8080/auth/realms/master/protocol/openid-connect/token]
```

As you can see it does not work for me. Some says we should use 8443, but Keycloak does not bind to 8443.

**Comment by stianst** [comment]
Following the instructions in the guide it works fine for me. Since you're getting a `HTTPS required` message I'm assuming you're not opening the admin console using `http://localhost:8080`, but rather with a some different URL?

**Comment by vincentbernat** [comment]
I am using `http://localhost:8080` (and localhost resolves to 127.0.0.1). I also have this in the logs:

```
2022-06-17 14:18:59,231 WARN  [org.keycloak.events] (executor-thread-0) type=LOGIN_ERROR, realmId=67efc184-e337-460e-93fe-af4732757d4a, clientId=null, userId=null, ipAddress=100.107.255.1, error=ssl_required
2022-06-17 14:19:03,867 WARN  [org.keycloak.events] (executor-thread-5) type=LOGIN_ERROR, realmId=67efc184-e337-460e-93fe-af4732757d4a, clientId=null, userId=null, ipAddress=100.107.255.1, error=ssl_required
```

However, if I add `--net=host`, it works.
